script.js

```diff
@@ -0,0 +1,24 @@
+function getTriangleArea(a, h) {
+	if (a <= 0) {
+		console.log('Nieprawidłowe dane');
+	}
+
+	if (h <= 0) {
+		console.log('Nieprawidłowe dane');
+	}
+
+	else {
+		return a*h/2;
+	}
+}
+	getTriangleArea (console.log( getTriangleArea(10,6) ));
+
+var triangle1Area = getTriangleArea(10, 15),
+	triangle2Area = getTriangleArea(22, 30),
+	triangle3Area = getTriangleArea(100, 15);
+
+	getTriangleArea (console.log( triangle1Area ));
```

Comment:
what you want to do here? you can function in function. function getTriangleArea  should contain at least two number arguments